### PR TITLE
Fix challenge detail screen

### PR DESCRIPTION
**Pull request by LieutenantRoger**
Issue: https://github.com/topcoder-platform/community-app/issues/4965

Dependency: https://github.com/topcoder-platform/topcoder-react-lib/pull/259

### CODE REVIEW

File: src/shared/components/challenge-detail/Header/ChallengeTags.jsx [hunk 1]
```diff
@@ -36,15 +36,15 @@ export default function ChallengeTags(props) {
   let EventTag;
   let TrackTag;
   switch (track) {
-    case COMPETITION_TRACKS.DATA_SCIENCE:
+    case COMPETITION_TRACKS.DS:
```
Comment (luizrrodrigues) [inline src/shared/components/challenge-detail/Header/ChallengeTags.jsx:39]: @LieutenantRoger Please make sure to update all ref to `COMPETITION_TRACKS` like you have done here.

ref. comment here: https://github.com/topcoder-platform/topcoder-react-lib/commit/d7e336ea391ebf3c950cf09f999a48a00db7402a#r42801128

File: src/shared/containers/challenge-detail/index.jsx [hunk 2]
```diff
@@ -213,21 +214,21 @@ class ChallengeDetailPageContainer extends React.Component {
...
-        !recommendedChallenges[recommendedTechnology]
-        || (
-          Date.now() - recommendedChallenges[recommendedTechnology].lastUpdateOfActiveChallenges
-          > 10 * MIN
-        )
-      )
-    ) {
-      getAllRecommendedChallenges(auth.tokenV3, recommendedTechnology);
-    }
+    // const recommendedTechnology = getRecommendedTags(challenge);
```
Comment (luizrrodrigues) [inline src/shared/containers/challenge-detail/index.jsx:217]: @sushilshinde We good to comment a lot of code block instead remove that?

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

File: src/shared/containers/challenge-detail/index.jsx (hunk 2 above)
Comment (sushilshinde) [reply src/shared/containers/challenge-detail/index.jsx:217]: yes, for right now.

**Review by luizrrodrigues — APPROVED** [review]

**Comment by luizrrodrigues** [comment]
@LieutenantRoger changed missing one: https://github.com/topcoder-platform/community-app/pull/5008/commits/7e0f01373dc1c2ab9ece71b9caf1b63c46553b50

**Pull request merged**